python/lsst/ts/dimm/controllers/base_dimm.py

```diff
@@ -48,6 +49,7 @@ def __init__(self, log):
             "altitude": 0.0,
             "azimuth": 0.0,
             "hrnum": 0,
+            "controller_running": False,
```

Comment:
Uhm, why do you need this new flag here? From the code bellow it seems like `controller_running = False` is equivalent to `status = DIMMStatus["INITIALIZED"]` and `controller_running = True` is equivalent to `status = DIMMStatus["RUNNING"]` .

I did just realized a problem in the implementation of the status flag. The original idea was that this would be a bitmap so we could flip bits on and off to indicate each status. 

For instance; you could have the status as INITIALIZED and RUNNING: `1<<1 | 1<<2`

and also INITIALIZED and RUNNING and ERROR: `1<<1 | 1<<2 | 1<<3`

But I just noticed that the implementation is not carrying it out like this. 

My suggestion is that you remove this flag and, for now, assume `controller_running = status ==  DIMMStatus["RUNNING"]`. Then later we fix the bitmap operation and, if you still think this status is needed, we add a new "CONTROLLERRUNNING" bit...

Reply:
I introduced the new flag to be able to query the DIMM server status. DIMMStatus["RUNNING"] is set already when the CSC connects to the DIMM hardware, which doesn't automatically mean that it actually is producing sensible data. Judging by that, the `controller_running = status == DIMMStatus["RUNNING"]` doesn't seem to add much to the existing code. But OK, I followed your suggestion.

Reply:
If you have a reason for having `controller_running` you should probably leave it then... and we update it when refactoring the use of the status bit mask.

Reply:
Actually I think we can use the DIMMStatus already, but the point at which is gets set to `RUNNING` should be changed. I'll implement that and will push the code so you can see if you agree with me.